osquery/utils/windows/shelllnk.cpp

```diff
@@ -0,0 +1,635 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+#include <osquery/logger/logger.h>
+#include <osquery/utils/conversions/join.h>
+#include <osquery/utils/conversions/tryto.h>
+#include <osquery/utils/conversions/windows/strings.h>
+#include <osquery/utils/conversions/windows/windows_time.h>
+#include <osquery/utils/windows/shellitem.h>
+
+#include <boost/algorithm/hex.hpp>
+namespace osquery {
+
+struct LinkFlags {
```

Comment:
Are these `struct LinkFlags`, and the structures below also defined in the header? You can remove these definitions from this `.cpp` implementation file.

Reply:
good point, removed from cpp file